autoscheduler/frontend/src/redux/actions/termData.ts

```diff
@@ -0,0 +1,108 @@
+/* These all have to be in the same file due to dependency cycles
```

Comment:
Any suggestions for the location + name for this? I don't think `actions/termData` is really accurate since it doesn't actually contain actions. Maybe `reducers/termDataTypes`?

Reply:
That makes more sense, since the other action type `interface`s are all in the `reducers` folder